tests/test_cli.py

```diff
@@ -15,3 +23,102 @@ def test_reporter(capsys, reporter):
     captured = capsys.readouterr()
     assert captured.out == ""
     assert captured.err == "my test\nanother test\n"
+
+
+def test_docs(runner, monkeypatch):
+    calls = []
+    monkeypatch.setattr('webbrowser.open', lambda *args: calls.append(args))
```

Comment:
For cases like this where you actually need to record calls, I'd use `pytest-mock` (I use `monkeypatch` for simple cases where I don't need to record things or mock complex stuff, and `mock` when I need to).

There, you can simply use `mocker.patch('webbrowser.open', autospec=True)` (IIRC) and then use `mocker.assert_called_once_with(...)`.

Reply:
Also, why not patch the `webbrowser` function we're actually calling? Right now the patching is relying on implementation internal stuff in `webbrowser.py`.

Reply:
Thanks for the hint with the mocker.

The idea of checking against `webbrowser.open` was to ensure that no other method of `webbrowser` would get called. But I can remove it